src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/EnrichEventService.kt

```diff
@@ -87,7 +85,6 @@ class EnrichEventService(
         .map { conviction -> conviction.convictionId }
     ).distinct()
 
-  @Cacheable(CacheConfiguration.INDUCTION_APPOINTMENT_CACHE_NAME)
```

Comment:
There's other resources relating to this cache that we should consider deleting:

 - The architecture decision doc
 - Maybe     APPLICATION_CACHE_INDUCTION_CRON in some helm files (?)
 - Is there not a test for it?